backend/tests/test_project.py

```diff
@@ -173,7 +173,107 @@ def test_get_specific_type(self):
         json_response = json.loads(response3.get_data(as_text=True))
         self.assertEqual(len(json_response), 2)
 
+    # test get endpoint by completion status
+    def test_get_projects_by_completion_status(self):
+        test_list = [True, False]
```

Comment:
is there a reason why you decided to make a boolean array?

Reply:
Not really, thought it would be easier to see the tested inputs when it comes to code review.

Reply:
Ok, this is more of a nitpick & stylistic preference but I think it would be more readable if you just used the Boolean values instead and I think it's more commonly done that way anyways so the code reader knows exactly what to expect

Reply:
Thanks for the suggestion Celine, and also apologies for the confusion. I have updated the boolean values accordingly